dashboard/common/utils.py

```diff
@@ -13,9 +13,11 @@
 def format_github_url_using_https(github_url: str):
     ssh_base = "git@"
     https_base = "https://"
-    # If the URL is formatted for SSH, convert, otherwise, do nothing
+    # If the URL is formatted for SSH, convert, otherwise, replace .git extension with ""
```

Comment:
This seems fine. 

I _think_ you could have avoided the else and just had it like this too since it's the same in both. But this would accomplish the same and you've already tested it out. Thanks!

```
github_url = github_url.replace(".git", "")
if ssh_base == github_url[:len(ssh_base)]:
    github_url = github_url.replace(":", "/").replace(ssh_base, https_base)
```